docs/source/sharing.rst

```diff
@@ -0,0 +1,106 @@
+Sharing
+=======
+
+Plone comes with a sophisticated user management system that allows to assign users and groups with global roles and permissions. Sometimes this in not enough though and you might want to give users the permission to access or edit a specific part of your website or a specific content object. This is where local roles (located in the Plone sharing tab) come in handy.
```

Comment:
I would start wording things like if the reader doesn't know anything or had any previous contact with Plone, the CMS. ;) BTW, it's ok for me now, only wanted to mention it.